src/python/pants/option/config.py

```diff
@@ -208,7 +208,7 @@ def options(self, section: str) -> List[str]:
 
   @abstractmethod
   def defaults(self) -> Mapping[str, Any]:
-    """All the DEFAULT values."""
+    """All the DEFAULT values (not interpolated)."""
```

Comment:
What does "not interpolated" mean here?

Reply:
For example, we won't expand the value `%s(build_root)`. I can clarify this in the followup that adds TOML support.